romancal/flatfield/flat_field.py

```diff
@@ -54,6 +54,11 @@ def do_flat_field(output_model, flat_model):
                     'shape as the science data')
         log.warning('Step will be skipped')
         output_model.meta.cal_step.flat_field = 'SKIPPED'
+    elif output_model.meta.exposure.type != "WFI_IMAGE":
+        # Check to see if attempt to flatten non-Image data
+        log.warning('Exposure type is '+output_model.meta.exposure.type+', which is not WFI_IMAGE')
```

Comment:
I'd suggest log.info instead of log warning since skipping the step is not an error.

Reply:
I agree. I would make it one line log message, e.g.
`log.info("Skipping flat field for spectral exposure."`

Reply:
Good catch! Changed.